packages/esm-patient-list-app/src/api/hooks.ts

```diff
@@ -134,11 +134,23 @@ export function usePatientListMembers(
   pageSize: number = 10,
   v: string = 'full',
 ) {
-  const swrResult = useSWR<FetchResponse<CohortResponse<OpenmrsCohortMember>>, Error>(
+  const { data, error, isLoading, mutate } = useSWR<FetchResponse<CohortResponse<OpenmrsCohortMember>>, Error>(
     `${cohortUrl}/cohortmember?cohort=${patientListUuid}&startIndex=${startIndex}&limit=${pageSize}&v=${v}&q=${searchQuery}`,
     openmrsFetch,
   );
-  return { ...swrResult, data: swrResult?.data?.data?.results };
+
+  // FIXME: This is a workaround for removing a patient from a list
```

Comment:
@dkayiwa I've ended up filtering the lists on the frontend to remove patients from the list if a cohort membership has an `endDate` property.